daphne_worker/src/lib.rs

```diff
@@ -198,7 +211,7 @@ pub const DEFAULT_RESPONSE_HTML: &str = "<body>Daphne-Worker</body>";
 
 static ISOLATE_STATE: OnceCell<DaphneWorkerIsolateState> = OnceCell::new();
 
-impl DaphneWorkerRouter {
+impl<'a> DaphneWorkerRouter<'a> {
```

Comment:
Compiler seems fine with an anonymous lifetime here. If you think a named lifetime would be more useful/readible, I'd suggest `'srv` for consistency with how we've named lifetimes elsewhere (`'srv` for the process; `'req` for the request).
```suggestion
impl DaphneWorkerRouter<'_> {
```

Reply:
Whoops, in fact we need an actual lifetime here.
```suggestion
impl<'srv> DaphneWorkerRouter<'srv> {
```